WebView/app/src/main/assets/main.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+function sendAndroidMessage() {
+	Weather.sendMessage("The weather in WebView City today is: " + document.getElementById("description").innerHTML);
+}
+
+function getData() {
+	fetch("https://gcoleman799.github.io/Asset-Loader/weather.json").then(function(resp) {
+		return resp.json();
+	})
+	.then(function(data) {
+	    var location = JSON.stringify(data.location).replace(/['"]+/g, '');
+	    var description = JSON.stringify(data.description).replace(/['"]+/g, '');
+	    var icon = JSON.stringify(data.icon).replace(/['"]+/g, '');
+	    document.getElementById("location").innerHTML = location;
```

Comment:
Can we use innerText or textContent instead of innerHTML?

Reply:
Ok